components/Chips/examples/ChipsSizingExampleViewController.m

```diff
@@ -29,10 +30,17 @@ - (void)viewDidLoad {
 
   self.view.backgroundColor = [UIColor whiteColor];
 
+  self.containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
This will overwrite any theme injected by the Catalog. Please move this to `init` or add a `nil` check before overwriting.